app/jobs/fetch_job.rb

```diff
@@ -46,25 +47,11 @@ def downloader_args
   end
   # rubocop:enable Metrics/AbcSize
 
-  def warcs?
```

Comment:
Most of these methods moved into the Registering module so they could be shared.